src/NuGet.Clients/VsExtension/.vsixinclude

```diff
@@ -46,3 +47,4 @@ NuGet.Core.dll
 # 3rd party dlls to keep
 Lucene.Net.dll
 Microsoft.Web.XmlTransform.dll
+Newtonsoft.Json.dll
```

Comment:
Talked to Rob and Justin about this. VS controls the version of Newtonsoft.Json that is loaded so this will be harmless. It is needed when we shell out to MSBuild to run our NuGet.Build.Tasks.dll (of which Newtonsoft.Json is a dependency).